packages/types/src/type/Vote.spec.ts

```diff
@@ -17,15 +19,76 @@ describe('Vote', () => {
     expect(new Vote(new Boolean(true)).isAye).toBe(true);
   });
 
-  it('has isYay for negative', () => {
+  it('has isNay for negative', () => {
     expect(new Vote(new Boolean(false)).isNay).toBe(true);
   });
 
-  it('is positive for negative numbers', () => {
+  it('is Aye for negative numbers', () => {
     expect(new Vote(-999).isAye).toBe(true);
```

Comment:
I would change 999 to something else: `new Vote(a)` -> `a` needs to be between -128 and 127, so 999 won't work.

Also, I would add another test: `new Vote(a).toNumber` should be equal to `a`

Reply:
> Also, I would add another test: new Vote(a).toNumber should be equal to a

should it? my understanding was that a positive vote is a positive vote, and the weight to it is only determined by the conviction. so toNumber should be -1 for `new Vote(a)`, `a > 0` and 0 for `new Vote(a)`, `a < 0` and `a ≠0`